qiskit/compiler/assembler.py

```diff
@@ -140,7 +141,11 @@ def assemble(
 
             ['gaussian', 'constant']
         init_qubits: Whether to reset the qubits to the ground state for each shot.
-                     Default: ``True``.
+            Default: ``True``.
+        use_measure_esp: Whether to use ESP (excited state promoted) readout for the final
```

Comment:
```suggestion
        use_measure_esp: Whether to use excited state promoted (ESP) readout for the final
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6543/commits/de6d6073315d5607ef59c929861b38aa6d47847a.